maven-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -19,7 +20,21 @@
   </properties>
 
   <dependencies>
-    <!-- TODO: Add any implementation-specific dependencies -->
+    <dependency>
+      <groupId>com.ozonehis</groupId>
+      <artifactId>ozone</artifactId>
+      <type>zip</type>
+      <version>${version}</version>
```

Comment:
I don't think this is the right value. It works for now, but that's because the version of Ozone is 1.0.0-SNAPSHOT. However, once we've released Ozone 1.3.0, presumably that's the version that people will want to start from.

Reply:
But this Maven Archetype version will always be equal to the Ozone version isn't it? As a Maven submodule of Ozone, the versions must match the root pom version, right?

Reply:
`${version}`, in the archetype POM corresponds to the version the user enters when running `mvn archetype:generate`... We could add a property here, though.